packages/libraries/main/src/types.ts

```diff
@@ -209,6 +224,10 @@ export interface OptionsType {
    * @default 256
    */
   maxLength?: number
+  /**
+   * @description Define the values to calculate the scoring and attack times. DO NOT CHANGE unless you know what you are doing. The default values are just fine.
```

Comment:
Consider expanding on this a little bit?
```
The default values are just fine as long as you are using a strong, slow hash function.
```
And maybe add a note about why these values are configurable:
```
Can be adjusted to account for increasingly powerful attacker hardware.
```